volatility3/framework/constants/__init__.py

```diff
@@ -40,7 +40,7 @@
 # We use the SemVer 2.0.0 versioning scheme
 VERSION_MAJOR = 2  # Number of releases of the library with a breaking change
 VERSION_MINOR = 1  # Number of changes that only add to the interface
-VERSION_PATCH = 0  # Number of changes that do not change the interface
+VERSION_PATCH = 1  # Number of changes that do not change the interface
```

Comment:
If there is a consensus on PR, I will revise the version according to the update details.